src/Appwrite/Platform/Modules/DevKeys/Http/DevKeys/ListKeys.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Appwrite\Platform\Modules\DevKeys\Http\DevKeys;
+
+use Appwrite\Extend\Exception;
+use Appwrite\Utopia\Response;
+use Utopia\Database\Database;
+use Utopia\Database\Document;
+use Utopia\Database\Query;
+use Utopia\Database\Validator\UID;
+use Utopia\Platform\Action;
+use Utopia\Platform\Scope\HTTP;
+
+class ListKeys extends Action
+{
+    use HTTP;
+    public static function getName()
+    {
+        return 'listKeys';
+    }
+
+    public function __construct()
+    {
+        $this
+            ->setHttpMethod(Action::HTTP_REQUEST_METHOD_GET)
+            ->setHttpPath('/v1/projects/:projectId/development-keys')
+            ->desc('List keys')
+            ->groups(['api', 'projects'])
+            ->label('scope', 'projects.read')
+            ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+            ->label('sdk.namespace', 'projects')
+            ->label('sdk.method', 'listKeys')
```

Comment:
listDevKeys, to be consistent with the rest.

Reply:
updated.